### PR TITLE
Task #11003: Add model location, travelling and Restful location

### PR DESCRIPTION
**Pull request by huynhlv**
Screenshot from 2019-04-22 19-41-27
LinkRedmi: https://edu-redmine.sun-asterisk.vn/issues/11003

### CODE REVIEW

**Comment by huynhlv** [comment]
ready

File: app/controllers/admin/locations_controller.rb [hunk 1]
```diff
@@ -0,0 +1,52 @@
+class Admin::LocationsController < ApplicationController
+  before_action :admin_user
+  before_action :load_location, except: %i(create index)
```
Comment (VuThiTranVan) [inline app/controllers/admin/locations_controller.rb]: thêm line trống trước line này

File: app/views/admin/locations/_location.html.erb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<tr>
+  <td><%= location.id %></td>
+  <td><%= location.name %></td>
+  <td>
+    <%= link_to t(".edit"), edit_admin_location_path(location) %>
+  </td>
+  <td>
+    <%= link_to t(".delete"), admin_location_path(location), method: :delete,
+        data: { confirm: t(".you_sure") } %>
```
Comment (VuThiTranVan) [inline app/views/admin/locations/_location.html.erb]: indent sai nhóe

File: app/views/admin/locations/_location.html.erb [hunk 3]
```diff
@@ -0,0 +1,11 @@
+<tr>
+  <td><%= location.id %></td>
```
Comment (VuThiTranVan) [inline app/views/admin/locations/_location.html.erb]: ko hiển thị id nhé trai

**Comment by huynhlv** [comment]
ready

**Pull request merged**